dev-python/flask-cors/flask-cors-9999.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+
+if [[ "${PV}" == "9999" ]]; then
+	EGIT_REPO_URI="https://github.com/wcdolphin/${PN}.git"
+	inherit git-r3
+else
+	MY_PN="Flask-Cors"
+	MY_P="${MY_PN}-${PV}"
+	SRC_URI="mirror://pypi/${MY_P:0:1}/${MY_PN}/${MY_P}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="A Flask extension for Cross Origin Resource Sharing (CORS)"
+HOMEPAGE="https://github.com/wcdolphin/flask-cors https://pypi.org/project/Flask-Cors/"
+
+LICENSE="MIT"
+SLOT="0"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-python/flask[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+BDEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${MY_P}"
+
+distutils_enable_tests nose
+distutils_enable_sphinx doc --no-autodoc
+
+python_prepare_all() {
+	distutils-r1_python_prepare_all
+	# test is failing with >flask-1.1, see #253 on gh
+	rm tests/decorator/test_exception_interception.py || die
+}
+
+python_test() {
+	nosetests -vv || die "Testing failed with ${EPYTHON}"
+}
```

Comment:
Why are we adding a live ebuild, especially when it's not maintained by you? Does something need this? Wouldn't beets-9999 work with latest flask-cor?

And on that regard, can we even maintain beets-9999?

Reply:
> Why are we adding a live ebuild, especially when it's not maintained by you?

When on package that don't have one, I always add the live-ebuild,
I think it is nice to have when working / debugging on the package.
Also it is good for user, when getting an issue, most upstream will ask them to test on master.

> Does something need this?

Beside maintainer no, and it was not existing before.

> Wouldn't beets-9999 work with latest flask-cor?

Beets even on master require (suggest / test) some package versions,
so using master dependencies could work (like newer python version),
but also could cause issue, and upstream may not want to support them.

> And on that regard, can we even maintain beets-9999?

I think I can, but I won't upgrade the live-ebuild on each master commit obviously,
mostly when bumping a new release.

Reply:
Please drop the -9999 ebuilds then. We've had enough feedback that we can't react to -9999 breakage fast enough (we, as in the proxy-maint project).

Reply:
Ok I am dropping the -9999 ebuild